src/example-types/esTwoLevelAggregation.js

```diff
@@ -30,19 +36,17 @@ module.exports = {
               context.key_data
             )
           ),
-          aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
-                _.extend(
-                  {
-                    field: context.value_field,
-                  },
+          aggs: F.arrayToObject(
+            _.identity,
+            metric =>
+              _.omitBy(_.isNil, {
```

Comment:
the omitBy + isNil should be on the value of the agg which is merging on `value_data`. Putting it here does nothing because there will always be value on `[metric]` and it is the only key on the object. While you're at it, you could factor out `omitNil` as per https://github.com/smartprocure/futil-js/issues/267

Reply:
Done